base.js

```diff
@@ -18,4 +18,12 @@ module.exports = {
     'plugin:@typescript-eslint/recommended',
     'plugin:eslint-comments/recommended',
   ],
+  rules: {
+    'no-warning-comments': [
+      'error',
+      {
+        terms: ['DEBUG'],
+      },
+    ],
+  },
```

Comment:
Could we put this in the `overrides` file? The naming might be a bit confusing but the idea of the file is to hold all the rules that we use explicitly. The `base.js` here is just for the basic stuff to get the right configs and what not. This way if a third party plugin is used for one of our configs we would put the overrides after which would lead to the rule using our setting rather than the one from the plugin.